rholang/src/main/scala/coop/rchain/rholang/interpreter/matcher/StreamT.scala

```diff
@@ -37,7 +38,7 @@ object StreamT extends StreamTInstances0 {
     StreamT(Monad[F].pure(SCons(a, empty)))
 
   def liftF[F[_]: Monad, A](fa: F[A]): StreamT[F, A] =
-    StreamT(fa.map(value => SCons(value, empty)))
+    StreamT(Monad[F].map(fa)(value => SCons(value, empty)))
```

Comment:
why not using `map` here? if you want to use that approach, then consider `Functor[A]`

Reply:
To avoid creation of a FunctorOps instance. I'll try sticking to the least powerful typeclass in all the methods, commit to follow soon.

Reply:
Done, back to you :)